### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by mcalus3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,40 +2,78 @@
...
-## Setup
+## Features
+
+- List Travis CI Builds
+- Retrigger builds
 
-1. If you have standalone app (you didn't clone this repo), then do
+## How to enable Travis-ci plugin in Backstage
+
+Travis-ci plugin is a built-in part of vanilla backstage project. To start using it for your component, you have to:
```
Comment (dtuite) [inline README.md]: > vanilla backstage project

The terminology the Backstage team use is "part of the Backstage sample app".

File: README.md [hunk 2]
```diff
@@ -2,40 +2,78 @@
...
+```yml
+travis-ci.com/repo-slug: <owner-name>/<project-name>
+```
+
+2. add your developer api key (from https://travis-ci.com/account/preferences) to the environmental variables for your backstage backend server:
+   `TRAVISCI_AUTH_TOKEN="token <your-api-key>"`
+
+## How to add Travis-ci project dependency to Backstage app
+
+If you have your own version of backstage without this plugin, here it's how to add it:
```
Comment (dtuite) [inline README.md]: > your own version of backstage

This should be "your own backstage application".

The terminology is that users of Backstage will use the @backstage/create-app CLI to create their own "Backstage application". If they just want to try out Backstage, then they may use the "Backstage sample app" which comes inside the Backstage repo.

You totally have the idea right, just slightly different terminology than the Backstage team seem to use.

File: README.md [hunk 3]
```diff
@@ -2,40 +2,78 @@
...
-builder.add(travisCIApiRef, new TravisCIApi());
+```tsx
+// packages/app/src/components/catalog/EntityPage.tsx
+case isTravisCIAvailable(entity):
+    content = <RecentTravisCIBuildsWidget entity={entity} />;
+    break;
 ```
 
-5. Run app with `yarn start` and navigate to `/travis-ci`
+## Develop plugin locally
```
Comment (dtuite) [inline README.md:68]: I like that this is here to help people contribute 👍 .

**Review by dtuite — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (iain-b) [reply README.md]: I think we need to make a similar distinction on the marketing site?

**Pull request merged**